NUlid/Ulid.cs

```diff
@@ -567,7 +586,11 @@ public override string ToString()
     /// </summary>
     /// <returns>A 16-element byte array.</returns>
     public byte[] ToByteArray()
-        => new byte[] { _a, _b, _c, _d, _e, _f, _g, _h, _i, _j, _k, _l, _m, _n, _o, _p };
+    {
+        var bytes = new byte[16];
+        Unsafe.WriteUnaligned(ref bytes[0], this);
```

Comment:
```suggestion
        Unsafe.WriteUnaligned(ref MemoryMarshal.GetArrayDataReference(bytes), this);
```
Avoids the bound-check that `bytes[0]` incurs.

Reply:
`MemoryMarshal.GetArrayDataReference` is not avaliable in .Net Standard 2.0. Would `MemoryMarshal.Write(bytes, ref this);` be equivalent?

Reply:
👍🏻 yes, that would be even the "safer" equivalent.